src/components/content/home/useListRegisteredServicesQuery.tsx

```diff
@@ -0,0 +1,15 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { useQuery } from '@tanstack/react-query';
+import { ServiceVendorService } from '../../../xpanse-api/generated';
+
+export default function useListRegisteredServicesQuery() {
```

Comment:
there is already a custom hook for this API - useAvailableServiceTemplatesQuery
Please see if we can reuse it.